_posts/0000-01-02-Rithwikperla.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)

Reply:
cool!!